src/utils/api.js

```diff
@@ -19,3 +19,11 @@ export const getArticle = (setArticle, article_id) => {
     }).catch((err) => {
     })
 }
+
+export const getComments = (setComments, article_id) => {
+    return api.get(`/articles/${article_id}/comments`)
+    .then(({data}) => {
+        setComments(data.comments)
+    }).catch((err) => {
```

Comment:
Please handle the setting of comments in the component. 
A function should only have on responsibility. 
The name of the function is `getComments` - thats all it should do. 

You can then setComments in the comments component. It also make it more reusable, as you might not always want to use the data to set something, you might want to compare it with something, or some other kind of data manipulation.